<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSimulatorXPlaneConfigWindow</class>
 <widget class="QWidget" name="CSimulatorXPlaneConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>585</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>X-Plane plugin settings</string>
  </property>
  <layout class="QGridLayout" name="gl_ConfigWindow">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="horizontalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gb_DBus">
     <property name="title">
      <string>xswiftbus DBus</string>
     </property>
     <layout class="QVBoxLayout" name="vl_DBus">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="swift::gui::components::CDBusServerAddressSelector" name="comp_SettingsXSwiftBus">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gb_XPSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_MaxAircraft">
        <property name="text">
         <string>Max.aircraft</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sb_MaxAircraft">
        <property name="maximum">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_MaxDistance">
        <property name="text">
         <string>Max.distance</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="ds_MaxDrawDistanceNM">
        <property name="suffix">
         <string>NM</string>
        </property>
        <property name="maximum">
         <double>250.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_FollowAircraftDistance">
        <property name="text">
         <string>Follow acft. distance</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="sb_FollowAircraftDistanceM">
        <property name="suffix">
         <string>m</string>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_Labels">
        <property name="text">
         <string>Labels</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="cb_DrawLabels">
        <property name="text">
         <string>draw labels</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbl_LabelColor">
        <property name="text">
         <string>Label color</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="swift::gui::components::CColorSelector" name="cs_LabelColor" />
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbl_BundleLights">
        <property name="text">
         <string>Bundle lights</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="cb_BundleTaxiLandingLights">
        <property name="text">
         <string>bundle taxi and landing lights</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lbl_TcasEnabled">
        <property name="text">
         <string>TCAS</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="cb_TcasEnabled">
        <property name="text">
         <string>show traffic on TCAS</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lbl_DebugMode">
        <property name="text">
         <string>Logging</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="cb_LogRenderPhases">
        <property name="text">
         <string>log.render phases</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="lbl_MessageBox">
        <property name="text">
         <string>Message box </string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QWidget" name="wi_MessageBox" native="true">
        <layout class="QHBoxLayout" name="hl_MessageBox">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="sb_MessageBoxLines">
           <property name="suffix">
            <string> lines</string>
           </property>
           <property name="minimum">
            <number>3</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="lbl_MessageBoxDuration">
        <property name="text">
         <string>Box disappears after</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QSpinBox" name="sb_MessageBoxDuration">
        <property name="suffix">
         <string>ms</string>
        </property>
        <property name="minimum">
         <number>1000</number>
        </property>
        <property name="maximum">
         <number>25000</number>
        </property>
        <property name="singleStep">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="lbl_MessageBoxMargins">
        <property name="text">
         <string>Box margins px.</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QWidget" name="wi_Margins" native="true">
        <layout class="QGridLayout" name="gl_MessageBoxMargins">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="2">
          <widget class="QLabel" name="lbl_MarginsLeft">
           <property name="text">
            <string>left</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_MsgBoxMarginsBottom">
           <property name="placeholderText">
            <string>bottom</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_MarginsTop">
           <property name="text">
            <string>top</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lbl_MarginsBottom">
           <property name="text">
            <string>bottom</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="le_MsgBoxMarginsTop">
           <property name="placeholderText">
            <string>top</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="le_MsgBoxMarginsRight">
           <property name="placeholderText">
            <string>right</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="le_MsgBoxMarginsLeft">
           <property name="placeholderText">
            <string>left</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="lbl_MarginsRight">
           <property name="text">
            <string>right</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="lbl_MessageColors">
        <property name="text">
         <string>Message colors</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QWidget" name="wi_MessageColors" native="true">
        <layout class="QGridLayout" name="gl_MessageColors">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="0"><widget class="swift::gui::components::CColorSelector" name="cs_ColorFreq" /></item>
         <item row="1" column="1"><widget class="swift::gui::components::CColorSelector" name="cs_ColorPriv" /></item>
         <item row="1" column="2"><widget class="swift::gui::components::CColorSelector" name="cs_ColorServ" /></item>
         <item row="1" column="3"><widget class="swift::gui::components::CColorSelector" name="cs_ColorStat" /></item>
         <item row="1" column="4"><widget class="swift::gui::components::CColorSelector" name="cs_ColorSup" /></item>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_ColorFreq">
           <property name="text">
            <string>Freq</string>
           </property>
           <property name="toolTip">
            <string>Frequency messages</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lbl_ColorPriv">
           <property name="text">
            <string>Priv</string>
           </property>
           <property name="toolTip">
            <string>Private messages</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lbl_ColorServ">
           <property name="text">
            <string>Serv</string>
           </property>
           <property name="toolTip">
            <string>Server messages</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="lbl_ColorStat">
           <property name="text">
            <string>Stat</string>
           </property>
           <property name="toolTip">
            <string>Status messages</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="lbl_ColorSup">
           <property name="text">
            <string>Sup</string>
           </property>
           <property name="toolTip">
            <string>Supervisor messages</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="lbl_NightTextureMode">
        <property name="text">
         <string>Night texture</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QComboBox" name="cb_NightTextureMode">
        <item>
         <property name="text">
          <string>auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>daylight texture only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>night texture only</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lbl_TerrainProbeEnabled">
        <property name="text">
         <string>Terrain probe</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="cb_TerrainProbeEnabled">
        <property name="text">
         <string>probe to determine ground elevation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="bb_OkCancel">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="vs_Buutons">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CDBusServerAddressSelector</class>
   <extends>QFrame</extends>
   <header>gui/components/dbusserveraddressselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CColorSelector</class>
   <extends>QFrame</extends>
   <header>gui/components/colorselector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sb_MaxAircraft</tabstop>
  <tabstop>ds_MaxDrawDistanceNM</tabstop>
  <tabstop>sb_FollowAircraftDistanceM</tabstop>
  <tabstop>cb_DrawLabels</tabstop>
  <tabstop>cs_LabelColor</tabstop>
  <tabstop>cb_BundleTaxiLandingLights</tabstop>
  <tabstop>cb_TcasEnabled</tabstop>
  <tabstop>cb_LogRenderPhases</tabstop>
  <tabstop>sb_MessageBoxLines</tabstop>
  <tabstop>le_MsgBoxMarginsTop</tabstop>
  <tabstop>le_MsgBoxMarginsBottom</tabstop>
  <tabstop>le_MsgBoxMarginsLeft</tabstop>
  <tabstop>le_MsgBoxMarginsRight</tabstop>
  <tabstop>cs_ColorFreq</tabstop>
  <tabstop>cs_ColorPriv</tabstop>
  <tabstop>cs_ColorServ</tabstop>
  <tabstop>cs_ColorStat</tabstop>
  <tabstop>cs_ColorSup</tabstop>
  <tabstop>cb_NightTextureMode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
